storage/client_test.go

```diff
@@ -51,6 +51,15 @@ import (
 	"github.com/gogo/protobuf/proto"
 )
 
+func rg1(s *storage.Store) client.Sender {
+	return client.Wrap(s, func(ba roachpb.BatchRequest) roachpb.BatchRequest {
+		if ba.RangeID == 0 {
+			ba.RangeID = 1
```

Comment:
By "on the Store itself" I meant `Store.Send`, not a new method like `Store.TestSender`. It would put code intended for tests in the production code path, but we already do something similar in LocalSender to look up missing store IDs.